### PR TITLE
Update it.json

### PR DESCRIPTION
**Pull request by Osrek**
Translation completed (I think)... will revise it soon, but it's good to go for now. Edited lines since some eng sentences don't make sense in it

### CODE REVIEW

**Comment by enricomg** [comment]
it was a draft to improve, sorry

File: js/web/_i18n/it.json [hunk 1]
```diff
@@ -71,239 +71,239 @@
...
-		"Boxes.OwnpartCalculator.PatronPart": "Patron share",
-		"Boxes.OwnpartCalculator.Place": "Place",
-		"Boxes.OwnpartCalculator.PlayerName": "Player", //Todo: Translate
-		"Boxes.OwnpartCalculator.Scheme": "Scheme", //Todo: Translate
-		"Boxes.OwnpartCalculator.Step": "Step",
-		"Boxes.OwnpartCalculator.Title": "Calcolatore di co-pagamento",
+		"Boxes.OwnpartCalculator.PatronPart": "Contributo posizioni",
+		"Boxes.OwnpartCalculator.Place": "Posto",
+		"Boxes.OwnpartCalculator.PlayerName": "Giocatore",
+		"Boxes.OwnpartCalculator.Scheme": "Schema", //Todo: no context
```
Comment (Th3C0D3R) [inline js/web/_i18n/it.json]: please remove `//Todo: no context` at this line to get approved

File: js/web/_i18n/it.json [hunk 2]
```diff
@@ -71,239 +71,239 @@
...
+		"Boxes.Units.NotFilled": "Non riempito",
 		"Boxes.Units.Proportionally": "Proportionally",
 		"Boxes.Units.Quantity": "Quantità",
 		"Boxes.Units.ReadyToLoot": "Pronto a saccheggiare!",
 		"Boxes.Units.Status": "Status",
 		"Boxes.Units.Title": "Panoramica dell'esercito",
-		"Boxes.Units.Unbind": "Unbound",
+		"Boxes.Units.Unbind": "Indipendenti",
 		"Boxes.Units.Unit": "Unità",
-		"DateTime": "D / M / YY h: mm: ss a",
```
Comment (Th3C0D3R) [inline js/web/_i18n/it.json]: Dont Change this, revert it to: "D / M / YY h: mm: ss a",

Comment (Osrek) [reply js/web/_i18n/it.json]: noticed it, I forgot the data wouldn't have changed

**Comment by enricomg** [comment]
line 230 to change from Acuq to Acqua

**Pull request merged**